src/docker-images/azure-blob-adapter/wsgi.py

```diff
@@ -21,6 +21,13 @@
 append_blob_service = AppendBlobService(connection_string=connection_string)
 
 
+def _get_blob_index(blob_name):
+    try:
+        return int(blob_name.split('.', 1)[1])
```

Comment:
I remember the `blob_name` has format `jobs.<job id>.<idx>`? If so, should this be `int(blob_name.split('.')[-1])`?

Reply:
Thanks for reminding, my bad. Fixed with `[2]`.

Retrieving the 3rd is for triggering the `IndexError` in case of there are only 2 items in the split list (the first blob is named `jobs.<job-id>`).

Reply:
Please add a test case for this.